site/content/docs/2.0/content/font.md

```diff
@@ -53,12 +44,17 @@ project in Creative Cloud Typekit and use it in all of their web projects.
 
 **Advantages**
 - You can tailor your font project to only include the fonts that your web site or app uses, minimizing the number of assets an end-user has to download.
-- Faster page load for users that have never visited other Arizona.edu websites that include the centrally managed font.
+- Faster page load if your site uses a smaller subset of font variants than what are included in the shared library.
 
 **Disadvantages**
-- Slower page load for users that have visited other Arizona.edu websites that include the centrally managed font.
-- You have to manage your own font in Typekit using a netid that is tied to one person, creating a single point of failure if that user leaves the university.
-
+- You have to manage your own font in Typekit using a NetID.
+  - If the netid is tied to one person, it creates a single point of failure if
+    that user leaves the University.
+  - If the NetID is a shared "test NetID" you have to manage credentials, and sign it
+    up to use Adobe products.
+
+**Note** If you choose to manage your own web font project, it is strongly recommended that you go through the effort to use a [test NetID](https://apps.iam.arizona.edu/).
```

Comment:
Didn't this used to say something about "shared by your department"?

```suggestion
**Note** If you choose to manage your own web font project, it is strongly recommended that you go through the effort to use a shared [test NetID](https://apps.iam.arizona.edu/).
```

Reply:
https://github.com/az-digital/arizona-bootstrap/commit/77591fe30e06042c3735c60572447974065f829b
removed the “, which is shared with your department” (but it would be good to have something about it being shared).